### PR TITLE
feat: e2e tests for nuxt

### LINKED ISSUE #1885: [FEATURE] Conduct E2E testing on nuxt files rather than Storybook
**Describe how this feature will help us improve**
Describe why do you think this feature is needed. 

Currently we are conducting automatic E2E tests in our pull requests as well as pushes. Previously we were doing that with Storybook, but recently we have added nuxt package with ready components. Converting E2E tests to test static Nuxt page rather than Storybook that needs to be built would save us a lot of minutes for Github Actions as well as fix the problem with tests failing due to timeout caused by Storybook built time.

**Is your proposal related to a problem?**
Describe the issue or type its number

**Describe the solution you'd considered**
A concise description of your preferred solution. Things to address include:

  - Details of the technical implementation
  - Tradeoffs made in design decisions
  - Caveats and considerations for the future

If there are multiple solutions, please present each one separately. Save comparisons for the very end.

**Additional context**
Anything you would like to add but was not sure which section it fits.

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue

Closes #1885 

# Scope of work

# Screenshots of visual changes

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/nuxt/layouts/default.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
         </SfButton>
       </template>
     </SfTopBar>
-    <SfHeader class="sf-header--multiline" title="Storefront UI" logo="/assets/logo.svg" cartItemsQty="1" is-sticky>
+    <SfHeader class="sf-header--multiline" title="Storefront UI" logo="/assets/logo.svg" cartItemsQty="1">
```
Comment (Baroshem) [inline packages/nuxt/layouts/default.vue:19]: Could you explain why there is a change like that? Was it breaking the E2E test?

File: packages/nuxt/package.json [hunk 2]
```diff
@@ -19,12 +21,16 @@
...
   "devDependencies": {
+    "@cypress/vue": "^2.2.3",
+    "@cypress/webpack-dev-server": "^1.4.0",
     "@vue/test-utils": "^1.1.3",
     "babel-core": "7.0.0-bridge.0",
     "babel-jest": "^26.6.3",
     "babel-loader": "^8.1.0",
     "babel-preset-vue": "^2.0.2",
     "css-loader": "^5.2.6",
+    "cypress": "^7.6.0",
```
Comment (Baroshem) [inline packages/nuxt/package.json:32]: Do we need both @cypress/vue and cypress packages?

File: packages/vue/src/stories/Introduction.stories.mdx [hunk 3]
```diff
@@ -220,7 +220,7 @@ import pkg from "../../package.json";
     </a>
   </li>
 </ul>
-<a href="/?path=/story/who-is-using-storefront-ui--page">Check out more!</a>
+<a href="?path=/story/who-is-using-storefront-ui--page">Check out more!</a>
```
Comment (Baroshem) [inline packages/vue/src/stories/Introduction.stories.mdx:223]: Good one! I forgot to add it in branch 0.10.9

**Review by Baroshem — APPROVED** [review]
I have few questions but overall I approve. Good job!

**Review by AdamPawlinski — APPROVED** [review]
It looks great for me. Super work :)

**Pull request merged**